autoload/codi/load.vim

```diff
@@ -37,7 +37,7 @@ let s:codi_default_interpreters = {
           \ 'prompt': '^\(>>>\|\.\.\.\) ',
           \ },
       \ 'javascript': {
-          \ 'bin': 'node',
+          \ 'bin': ['node','-e','require("repl").start({ignoreUndefined:true})'],
```

Comment:
Minor style gripe: please put spaces after commas and the colon after `ignoreUndefined`.